tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -714,6 +739,26 @@ public function userEnablesSyncOfShareUsingTheGraphApi(string $user, string $sha
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user tries to enable share sync of non-existing resource using the Graph API
```

Comment:
```suggestion
	 * @When user :user tries to enable share sync of a non-existent resource using the Graph API
```

https://www.oxfordlearnersdictionaries.com/definition/english/non-existent

non-existent is the adjective.

(maybe we have written non-existing in a lot of other places? we don't have to adjust all those right now!)